config/samples/flows_v1beta2_flowcollector.yaml

```diff
@@ -36,10 +36,12 @@ spec:
       server:
         port: 9102
       disableAlerts: []
-      includeList:
-        - node_ingress_bytes_total
-        - workload_ingress_bytes_total
-        - namespace_flows_total
+      # includeList:
+      #   - "node_ingress_bytes_total"
+      #   - "workload_ingress_bytes_total"
+      #   - "namespace_flows_total"
+      #   - "namespace_drop_packets_total"
+      #   - "namespace_rtt_seconds"
```

Comment:
@jotak can you explain this change? we've already started updating our CRDs to include those three defaults

Reply:
yes there is a subtlety. The default metrics are still these ones, but in CRD, the default is to not have the `includeList` specified. The reason is that, when you set values in includeList, even just keeping this exact same default metrics list, it's considered as a setting explictly set. In a following release, imagine that we add new metrics to the default list: people starting from scratch would get this default, but people upgrading would have the explicit list in settings, hence they would not get them.
And in addition it is also fixing a problem when converting from beta2 to beta1 (NETOBSERV-1405) , where setting default values in this sample was preventing to get updated defaults.

Reply:
but you can leave them in your CR, it's still the defaults metrics that we expect, and they are defined in code: https://github.com/netobserv/network-observability-operator/blob/main/pkg/metrics/predefined_metrics.go#L37-L42